---
title: "Tool: Mutect2"
output:
  github_document:
    toc: true
    toc_depth: 2
    fig_width: 5
    fig_height: 5
    dev: jpeg
---

```{r include = FALSE}
require(tidyverse); require(rmarkdown); require(knitr); require(sevenbridges); 

options(max.print = "99"); opts_knit$set(width = 99)
opts_chunk$set(cache=FALSE, tidy=FALSE)

```




https://igor.sbgenomics.com/u/dave/cgrwgs/apps/#dave/cgrwgs/mutect2

## Inputs



```{sh eval=FALSE}

BARCODE=$1 
PART=$2
NORMAL_BAM_NAME=$3
TUMOR_BAM_NAME=$4
DIR_VCF=$5

GLB_DIR_REF=/DCEG/Projects/PopulationExome/EAGLE/b37
DBSNP=${GLB_DIR_REF}/dbsnp_138.b37.vcf
COSMIC=${GLB_DIR_REF}/b37_cosmic_v54_120711.vcf
REFERENCE_GENOME=${GLB_DIR_REF}/human_g1k_v37_decoy.fasta

BED_DIR=/DCEG/Projects/Exome/SequencingData/variant_scripts_ji_prototyping_wgs_REBC/variant_calling_intervals
BAM_NORMAL=${NORMAL_BAM_NAME}
BAM_TUMOR=${TUMOR_BAM_NAME}
OUT_VCF=${DIR_VCF}/${BARCODE}.${PART}
	
BED_FILE=${BED_DIR}/variant_calling_intervals.${PART}.intervals
module load jdk
date

```

